includes/migrations/class-wc-stripe-restore-adaptive-pricing-after-amount-mismatch-update.php

```
@@ -0,0 +1,69 @@
+
+		if ( false === $previous_version ) {
+			update_option( self::MIGRATION_FLAG_OPTION, 'yes' );
+			return;
+		}
+
+		if ( 'yes' !== get_option( self::AMOUNT_MISMATCH_OPTION, 'no' ) ) {
+			update_option( self::MIGRATION_FLAG_OPTION, 'yes' );
+			return;
+		}
+
+		$stripe          = WC_Stripe::get_instance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe' in method 'maybe_migrate'. (undefined)

(StaticAccess)